Dockerfile

```diff
@@ -8,23 +8,26 @@
 # If you have a production use-case for phoenix, please get in touch!
 
 # Use an official Python runtime as a parent image
-FROM python:3.10
+FROM python:3.10-slim
 
 # Install nodejs
-RUN curl -fsSL https://deb.nodesource.com/setup_20.x | bash -
-RUN apt-get install -y nodejs
+RUN apt-get update && apt-get upgrade -y &&\
+    curl -fsSL https://deb.nodesource.com/setup_20.x | bash - &&\
+    apt-get install -y nodejs npm
 
 # Set the phoenix directory in the container to /phoenix
 WORKDIR /phoenix
 
 # Add the current directory contents into the container at /phoenix
 ADD . /phoenix
 
+
 # Install the app by building the typescript package
-RUN cd /phoenix/app && npm install && npm run build
+RUN cd /phoenix/app && npm install && npm run build && rm -rf /phoenix/app &&\
+    apt-get purge nodejs npm -y
```

Comment:
Suggestion: Use a two-stage build here instead of purging.

Reply:
Good call, I couldn't remember the name